### PR TITLE
[WIP] Keyboard tests

### PR DESCRIPTION
**Pull request by Owlbertz**
Now that we are possible to easily mock keyboard events, it's time to test the keyboard event handlers.

As the title says this is still work in progress, but feedback is wanted.

### CODE REVIEW

File: test/javascript/components/slider.js [hunk 1]
```diff
@@ -163,14 +163,77 @@ describe('Slider', function() {
...
+        initialStart: 10,
+        step: 1
+      });
+
+      plugin.$handle.focus()
+        .trigger(window.mockKeyboardEvent('ARROW_DOWN'));
+
+      plugin.$handle.should.have.attr('aria-valuenow', (10 - 1).toString());
+    });
+    it('decreases value by step size times 10 on SHIFT_AARROW_RIGHT', function() {
```
Comment (kball) [inline test/javascript/components/slider.js]: Slight typo

**Comment by kball** [comment]
YES! This will be hugely helpful

**Comment by Owlbertz** [comment]
Fixed an issue with https://github.com/zurb/foundation-sites/pull/9971/commits/53b82e1e11d4020a788a005592cdeb52e2af08d7.

@kball As far as I see there are now keyboard tests for every plugin that uses the Keyboard Util.
Do you agree that this make the PR merge-able?

**Comment by kball** [comment]
This is excellent. Nice work @Owlbertz

**Pull request merged**